app/scripts/views/complete_sign_up.js

```diff
@@ -94,9 +96,11 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
               return false;
             }
 
-            return self.getAccount().isSignedIn()
+            var account = self.getAccount();
+            return account.isSignedIn()
               .then(function (isSignedIn) {
                 if (isSignedIn) {
+                  self.interTabSend(EVENTS.SIGNIN_SUCCESS, account);
```

Comment:
> I thought it meant that the user will not be signed in

By "the user will not be signed in", I really meant "the user will not end up on the settings page".